src/chigame/games/models.py

```diff
@@ -6,13 +6,93 @@
 
 class Game(models.Model):
     """
-    A game (Chess, Checkers, etc.)
+    A game like Chess, Checkers, etc.
     """
 
-    name = models.TextField()
+    # Basic information
+    name = models.CharField(max_length=255)
     description = models.TextField()
+    image = models.URLField(default="/static/images/no_picture_available.png")
+    year_published = models.PositiveIntegerField(null=True)
+
+    # Gameplay information
+    rules = models.TextField(null=True)
+
     min_players = models.PositiveIntegerField()
     max_players = models.PositiveIntegerField()
+    suggested_age = models.PositiveSmallIntegerField(null=True)
+
+    expected_playtime = models.PositiveIntegerField(null=True)
+    min_playtime = models.PositiveIntegerField(null=True)
+    max_playtime = models.PositiveIntegerField(null=True)
+
+    complexity = models.PositiveSmallIntegerField(null=True)
+    category = models.ManyToManyField("Category", related_name="games")
+    mechanics = models.ManyToManyField("Mechanic", related_name="games")
+
+    # BGG information
+    BGG_id = models.PositiveIntegerField(unique=True, null=True)
```

Comment:
Another very small change, but I think this should not be unique just in case we end up supporting multiple versions of the same game

Reply:
Good suggestion!